sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -228,7 +228,7 @@ export interface PeekMessagesOptions extends OperationOptions {
 // @public
 export interface QueueDescription {
     authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
+    autoDeleteOnIdleInSeconds?: number;
     deadLetteringOnMessageExpiration?: boolean;
     defaultMessageTtl?: string;
```

Comment:
The PR description says that we are doing this type change for the below, but the actual PR makes the change only for autoDeleteOnIdle
- autoDeleteOnIdle
- defaultMessageTtl
- duplicateDetectionHistoryTimeWindow
- lockDuration

Reply:
I had updated only one of the attributes to show how the changes look like, mentioned that in the description :)
Waiting for some reviews to get some validation.

Will do the same for the rest of the attributes and the other option bags TopicDescription and SubscriptionDescription.

Reply:
Done!